packages/lib-user/src/hooks/usePanoptesProjects.js

```diff
@@ -27,11 +27,6 @@ async function fetchProjects({ query, token }) {
       })
 
       const { meta, projects } = response?.body || {}
-  
-      if (meta?.projects?.page_count > 5) {
-        console.warn('There are more than 500 projects related to this request. This is likely an error.')
-        return []
-      }
```

Comment:
This was added in [comment](https://github.com/zooniverse/front-end-monorepo/pull/5926/#discussion_r1504580196) and I'm intentionally removing it. We can no longer assume `page_size=100` in this custom hook.